docs/advisories/README.md

```diff
@@ -0,0 +1,6 @@
+## kops Advisories
+
+- [CVE 2017 14491](cve_2017_14491.md)
+- [CVE 2017 5753](meltdown-spectre.md)
+- [CVE 2017 5754](meltdown-spectre.md)
+- [Meltdown and Spectre](meltdown-spectre.md)
```

Comment:
One line per vuln methinks, no matter how many names they give it.  Should make life easier for our users:

`- [Meltdown and Spectre](meltdown-spectre.md) CVE-2017-5753, CVE-2017-5754, CVE-2017-5715`

Reply:
I would rather list each cve individually. More readable to me :man_shrugging:

Reply:
But for our users, they'll have to click through all 3 to see that they're one and the same.

Also, when we have another patch for the same CVE, what do we do then?